storage/innobase/os/os0file.cc

```diff
@@ -675,30 +681,58 @@ os_file_create_subdirs_if_needed(
 @param[in]	request	The IO context and type
 @return the number of bytes read/written or negative value on error */
 ssize_t
-SyncFileIO::execute(const IORequest& request)
+SyncFileIO::execute_low(const IORequest& request, ssize_t n)
```

Comment:
The added parameter is  not documented in the comment. I don’t see much value in duplicating the comment. There already is one at the member function declaration. Let’s use the common formatting style in functions that are almost completely rewritten:
```c++
ssize_t SyncFileIO::execute_low(const IORequest &request, ssize_t n)
{
  ut_ad(n > 0);
  ut_ad(size_t(n) <= os_file_request_size_max);
  if (request.is_read())
    return IF_WIN(tpool::,)pread(m_fh, m_buf, n, m_offset);
  else
    return IF_WIN(tpool::,)pwrite(m_fh, m_buf, n, m_offset);
}
```